spec.md

```diff
@@ -499,6 +499,55 @@ the module.  The `c`{.firrtl} sub-field contained in the `b`{.firrtl} sub-field
 flows into the module, and the `d`{.firrtl} sub-field contained in the
 `b`{.firrtl} sub-field flows out of the module.
 
+## Type Modifiers
+
+### Constant Type
+
+A constant type is a type whose value is guaranteed to be a compile-time 
+constant.  Constant types may be used in ports, wire, nodes, and generally 
```

Comment:
"whose value is guaranteed to never change during the execution of the circuit."

or

"whose value is guaranteed to be constant in the final physical circuit"

Since the value can actually change _after_ the firrtl compilation, when you link together multiple Verilog files.

Reply:
Hopefully clarified.